### DISCUSSION TITLE
Docker image eclipse-temurin alpine support for arm64?

### DISCUSSION THREAD

**Discussion by mholzer85** [adoptium/adoptium-support#741] [Q&A]
Hi!

There are docker images for eclipse-temurin for amd64 and arm64 (among others).
There's also an eclipse-temurin image based on alpine for amd64, but none based on alpine for arm64. Is there a specific reason for that?

Since alpine itself exists for arm64 as well...

We're currently using eclipse-temurin:11-alpine as base image for our services on production but we would need arm64 support as well. Is there anything planned in that direction?

Thanks in advance

**Comment by gdams** [comment]
duplicate of https://github.com/adoptium/containers/issues/158
